dashboard/test/lib/services/lti_test.rb

```diff
@@ -277,23 +277,28 @@ class Services::LtiTest < ActiveSupport::TestCase
   end
 
   test 'should add or remove student users when syncing a section' do
-    lti_integration = create :lti_integration
-    lti_course = create :lti_course, lti_integration: lti_integration
-    lti_section = create :lti_section, lti_course: lti_course
+    auth_id = "#{@lti_integration[:issuer]}|#{@lti_integration[:client_id]}|user-id-1"
+    user = create :teacher
+    create :lti_authentication_option, user: user, authentication_id: auth_id
```

Comment:
Theres a [trait](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/factories/factories.rb#L447) you can use too, `user = create :teacher, :with_lti_auth`.

Examples of using this trait are over in the [lti policies test](https://github.com/code-dot-org/code-dot-org/blob/07c28b8db120d8dbbb43e73a0f165bb2e6773b33/dashboard/test/lib/policies/lti_test.rb#L59)

Reply:
Although since you're instantiating the auth option with a specific auth id, you might have to do something like `user = create :teacher, :with_lti_auth, authentication_id: auth_id`, but I can't say for sure if that would work.

Reply:
Yeah... I'm actually not sure it will work. Might just be worth leaving it like you have it for now.

Reply:
I thought maybe this would work, but it sounds like it doesn't `user = create :teacher, :with_lti_auth, authentication_id: auth_id`

Reply:
I think it won't work because the trait is for a `user`, and `:authentication_id` is a property of the `authentication_option` not the `user`